pkg/auto/lights/patch.go

```diff
@@ -157,6 +157,11 @@ type source struct {
 
 func (b *BrightnessAutomation) processConfig(ctx context.Context, cfg config.Root, sources []*source, changes chan<- Patcher) (sourceCount int) {
 	logger := b.logger.With(zap.String("auto", cfg.Name))
+
+	if cfg.OnProcessError.BackOffMultiplier.Duration.Nanoseconds() <= 0 {
+		cfg.OnProcessError.BackOffMultiplier.Duration = config.DefaultBackOffMultiplier
+	}
```

Comment:
Need the same kind of check for `cfg.RefreshEvery` - don't want this to be negative or 0 either.